frontEnd/src/app/core/interfaces/auth/login-request.ts

```diff
@@ -0,0 +1,4 @@
+export interface IAuthRequestInput {
```

Comment:
interfaces should not be in **core** folder.
move **interfaces** folder to a folder like **src/app/shared/interfaces**.